### PR TITLE
Fix optional parameter issue for severless

### PR DESCRIPTION
**Pull request by lquanx**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -182,9 +182,9 @@ then
 fi
 
 #Default values
-issuerEndpoint=""
-oAuth2ApiEndpoint=""
-patientPickerEndpoint=""
+issuerEndpoint="undefined"
+oAuth2ApiEndpoint="undefined"
+patientPickerEndpoint="undefined"
```
Comment (lquanx) [inline scripts/install.sh:187]: serverless cannot handle empty string properly.

File: scripts/install.sh [hunk 2]
```diff
@@ -308,7 +308,7 @@ fi
 
 echo -e "\n\nFHIR Works is deploying. A fresh install will take ~20 mins\n\n"
 ## Deploy to stated region
-yarn run serverless-deploy --region $region --stage $stage --issuerEndpoint "$issuerEndpoint" --oAuth2ApiEndpoint "$oAuth2ApiEndpoint" --patientPickerEndpoint "$patientPickerEndpoint" || { echo >&2 "Failed to deploy serverless application."; exit 1; }
```
Comment (lquanx) [inline scripts/install.sh:311]: revert the change, so that it is easier to merge with upstream code

File: serverless.yaml [hunk 3]
```diff
@@ -12,9 +12,9 @@ custom:
   exportRequestTableJobStatusIndex: 'jobStatus-index'
   stage: ${opt:stage, self:provider.stage}
   region: ${opt:region, self:provider.region}
-  issuerEndpoint: ${opt:issuerEndpoint, ''}
-  oAuth2ApiEndpoint: ${opt:oAuth2ApiEndpoint, ''}
-  patientPickerEndpoint: ${opt:patientPickerEndpoint, ''}
```
Comment (lquanx) [inline serverless.yaml:17]: revert the change for easier merge with upstream code

**Review by lwarren01 — APPROVED** [review]
